packages/expo-updates/android/src/main/java/expo/modules/updates/loader/FileDownloader.java

```diff
@@ -274,13 +275,10 @@ private static Request setHeadersForUrl(Uri url, UpdatesConfiguration configurat
     return requestBuilder.build();
   }
 
-  /* package */ static Request setHeadersForManifestUrl(UpdatesConfiguration configuration, Context context) {
+  /* package */ static Request setHeadersForManifestUrl(UpdatesConfiguration configuration, JSONObject extraHeaders, Context context) {
     Request.Builder requestBuilder = new Request.Builder()
             .url(configuration.getUpdateUrl().toString())
             .header("Accept", "application/expo+json,application/json")
-            .header("Expo-Platform", "android")
-            .header("Expo-Api-Version", "1")
-            .header("Expo-Updates-Environment", "BARE")
             .header("Expo-JSON-Error", "true")
             // as of 2020-11-25, the EAS Update alpha returns an error if Expo-Accept-Signature: true is included in the request
             .header("Expo-Accept-Signature", String.valueOf(configuration.usesLegacyManifest()));
```

Comment:
Seems like it'd be nice to keep the Expo-specific headers together -- is there a clean way to do that across platforms?